src/Component/JsonSchema/Generator/Runtime/data/Normalizer/ValidatorTrait.php

```diff
@@ -6,7 +6,7 @@ trait ValidatorTrait
 {
     protected function validate(array $data, Constraint $constraint): void
     {
-        $validator = \Symfony\Component\Validator\Validation::createValidator();
+        $validator = Symfony\Component\Validator\Validation::createValidator();
```

Comment:
Won't work, see last comment.

Reply:
Will look into this.
Strange; these changes were made by the PHP CS Fixer;

Reply:
Should I commit the ....\expected\Runtime\.... files with my PR, guess not, right?
Just the change in the Generator class and the CHANGELOG.md.